api/rbody/rbody_test.go

```diff
@@ -96,45 +103,34 @@ func BenchmarkSeriesJson(b *testing.B) {
 	b.SetBytes(int64(len(pA) * 12))
 
 	b.ResetTimer()
+	var resp *JsonResponse
 	for n := 0; n < b.N; n++ {
-		js := bufPool.Get().([]byte)
-		js, err := graphiteRaintankJSON(js, data)
-		if err != nil || len(js) < 1000 {
-			panic(err)
-		}
-		bufPool.Put(js[:0])
+		resp = NewJsonResponse(200, models.SeriesByTarget(data), "")
+		resp.Body()
+		resp.Close()
 	}
 }
 
-func BenchmarkHttpRespJson(b *testing.B) {
+func BenchmarkHttpRespFastJson(b *testing.B) {
 	pA := make([]schema.Point, 1000, 1000)
-	pB := make([]schema.Point, 1000, 1000)
 	baseTs := 1500000000
 	for i := 0; i < 1000; i++ {
 		pA[i] = schema.Point{Val: float64(10000 * i), Ts: uint32(baseTs + 10*i)}
-		pB[i] = schema.Point{Val: 12.34 * float64(i), Ts: uint32(baseTs + 10*i)}
 	}
-	data := []Series{
+	data := []models.Series{
 		{
 			Target:     "some.metric.with.a-whole-bunch-of.integers",
 			Datapoints: pA,
 			Interval:   10,
 		},
-		{
-			Target:     "some.metric.with.a-whole-bunch-of.floats",
-			Datapoints: pB,
-			Interval:   10,
-		},
```

Comment:
why remove this? I'm guessing to make the workload the same as in the other test above.  that said, it would be great if the workload was the same as in previous versions of the code, so we could clearly see the benchmark performance before and after.  like i said elsewhere though I don't think there's a need for "the not fast version", so we can remove the above bench, and keep this one comparable to the previous one